src/core/config/__tests__/CustomModesManager.test.ts

```diff
@@ -1,3 +1,5 @@
+// npx jest src/core/config/__tests__/CustomModesManager.test.ts
```

Comment:
I've been running Windows in order to dogfood Roo Code on a new platform, and noticed a few tests failing on windows due to assumptions about the filesystem.